lib/structures/Message.js

```diff
@@ -506,7 +516,7 @@ class Message extends Base {
             "reactions",
             "referencedMesssage",
             "roleMentions",
-            "stickers",
```

Comment:
Still keep this (we will remove it when the whole prop gets removed in a future API version)